Build/start.sh

```diff
@@ -48,20 +48,21 @@ PREFERRED_RPROVIDER_wpa-supplicant = "sterling-supplicant-lwb"
 PREFERRED_RPROVIDER_wpa-supplicant-cli = "sterling-supplicant-lwb"
 PREFERRED_RPROVIDER_wpa-supplicant-passphrase = "sterling-supplicant-lwb"
 PREFERRED_RPROVIDER_wireless-regdb-static = "wireless-regdb"
+
+MACHINE_FEATURES_append = " docker"
+DISTRO_FEATURES_append = " virtualization"
 EOT
 
 #addition of meta-mistysom & mistylwb5p layers to bblayers.conf
 sed -i 's/renesas \\/&\n'\
 '  ${TOPDIR}\/..\/meta-mistysom \\\n'\
 '  ${TOPDIR}\/..\/meta-econsys \\\n'\
-'  ${TOPDIR}\/..\/meta-mistylwb5p\/meta-summit-radio-pre-3.4 \\\n'\
-'  ${TOPDIR}\/..\/meta-openembedded\/meta-networking \\'\
+'  ${TOPDIR}\/..\/meta-mistylwb5p\/meta-summit-radio-pre-3.4 \\'\
 '/' ${WORK}/build/conf/bblayers.conf
 
 # Disable recipes, tried BBMASK but was not working
 rm -rf ${WORK}/meta-mistylwb5p/meta-summit-radio-pre-3.4/recipes-packages/openssl
 rm -rf ${WORK}/meta-mistylwb5p/meta-summit-radio-pre-3.4/recipes-packages/summit-*
-rm -rf ${WORK}/meta-virtualization
```

Comment:
Can you give me some context on these changes, please?

Reply:
Line 58 is being deleted because it is conditionally being added in `bblayers.conf` depending on the existence of the virtualization flag. So we don't want a duplicate.
Line 64 is being deleted because the virtualization layer was disabled/deleted and now it will be included.

Reply:
Thanks!